src/scripts/main.js

```diff
@@ -1,7 +1,40 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const page = document.querySelector('body');
+  const message = document.createElement('div');
+
+  message.style.cssText = ` margin-top: ${posTop}px; 
+    margin-left: ${posRight}px`;
+
+  if (type === 'error') {
+    message.className = 'notification error';
+  }
+
+  if (type === 'success') {
+    message.className = 'notification success';
+  }
+
+  if (type === 'warning') {
+    message.className = 'notification warning';
+  }
```

Comment:
```suggestion
     message.className = `notification ${type}`;
```
you can make it in a more simple way 😉